demaceo/crate-1
# Feature/style survey

#### What does  this PR do?
- Creates the style survey that is used to collect style interest from a user
#### Where should the reviewer start?
- Preferences.js
#### How should this be manually tested?
- Clone down the repo
- `cd` into the api directory
- Run `npm start'
- In a separate terminal window
- `cd` into the web directory
- Run `npm start`
- Navigate to the local host url provided in the terminal
- Login as a user
- Click the "Crates" tab at the top of the page
- Click the "Subscribe+" button on any of the crates on the page
- A user will be brought to a page with the style survey
- The user can select photos that they like by clicking the radio buttons below each respective photo
- At the end of the survey is a 'submit' button
- Open the dev tools and then click the button and observe in the console how the styles have been collected into an array
#### Any background context you want to provide?
- We tried for an eternity to get the "POST" request working to push the selected styles to the API but sadly, we were getting 400 errors returned to us and we just couldn't get the fetch requests to work properly
